compiler+runtime/src/cpp/jank/runtime/core/math.cpp

```diff
@@ -546,6 +546,30 @@ namespace jank::runtime
       r);
   }
 
+  object_ptr quot(object_ptr const l, object_ptr const r)
+  {
+    return visit_number_like(
+      [](auto const typed_l, auto const r) -> object_ptr {
+        return visit_number_like(
+          [](auto const typed_r, auto const typed_l) -> object_ptr {
+            auto const typed_l_data{ to_number(typed_l) };
+            auto const typed_r_data{ to_number(typed_r->data) };
+#pragma clang diagnostic push
+#pragma clang diagnostic ignored "-Wfloat-equal"
+            if (typed_r_data == 0LL) {
+#pragma clang diagnostic pop
+              return make_box(-1); // TODO - throw exception
```

Comment:
In `clj` this throws a `ArithmeticException`, how do I throw exceptions from the c++ side?

Reply:
Just throw an `object_ptr`. jank has no base exception type. Something like `throw erase(make_box("Illegal divide by zero in 'quot'"));`